protobuf/common/error.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package common;
+
+option go_package = "github.com/layer5io/meshkit/protobuf/common;common";
+
+message CommonError {
```

Comment:
Just `Error`? It's `common` as it is in a package called `common`, isn't it?
Will there be other types of errors, i.e. not `Common`, here in the `common` package?

Reply:
The redundancy is mandatory here as if we use just `Error` in common package then you cannot use `Error` in any package that imports this package. @mgfeller